src/sapcc_sentrylogger/paste.py

```diff
@@ -0,0 +1,7 @@
+from sentry_sdk.integrations.wsgi import SentryWsgiMiddleware
+
+def sapcc_sentry_filter_factory(global_conf, **kwargs):
```

Comment:
There's a newline missing. Double newlines between imports and functions and further functions.